roda-core/roda-core/src/main/java/org/roda/core/plugins/plugins/common/AbstractConvertPlugin.java

```diff
@@ -243,173 +243,182 @@ protected Report executeOnAIP(IndexService index, ModelService model, StorageSer
+            if (!alteredFiles.isEmpty() && !createDIP) {
+              createNewFilesOnRepresentation(storage, model, unchangedFiles, newRepresentationID, notify);
+            }
 
-        } catch (RuntimeException | NotFoundException | GenericException | RequestNotValidException
-          | AuthorizationDeniedException | IOException | AlreadyExistsException e) {
-          LOGGER.error("Error processing AIP " + aip.getId() + ": " + e.getMessage(), e);
-          pluginResultState = PluginState.FAILURE;
-          reportState = pluginResultState;
-          reportItem.setPluginState(pluginResultState).setPluginDetails(e.getMessage());
-        } finally {
-          report.addReport(reportItem);
-          PluginHelper.updatePartialJobReport(this, model, reportItem, true, job);
-
-          if (!createDIP) {
-            try {
-              Representation rep = model.retrieveRepresentation(aip.getId(), newRepresentationID);
-              createPremisSkeletonOnRepresentation(model, aip.getId(), rep);
-            } catch (RequestNotValidException | GenericException | NotFoundException | AuthorizationDeniedException
-              | ValidationException | IOException | XmlException e) {
-              LOGGER.error("Error running premis skeleton on new representation: {}", e.getMessage());
+          } catch (RuntimeException | NotFoundException | GenericException | RequestNotValidException
+            | AuthorizationDeniedException | IOException | AlreadyExistsException e) {
+            LOGGER.error("Error processing AIP " + aip.getId() + ": " + e.getMessage(), e);
+            pluginResultState = PluginState.FAILURE;
+            reportState = pluginResultState;
+            reportItem.setPluginState(pluginResultState).setPluginDetails(e.getMessage());
+          } finally {
+            report.addReport(reportItem);
+            PluginHelper.updatePartialJobReport(this, model, reportItem, true, job);
+
+            if (!createDIP) {
+              try {
+                Representation rep = model.retrieveRepresentation(aip.getId(), newRepresentationID);
+                createPremisSkeletonOnRepresentation(model, aip.getId(), rep);
+              } catch (RequestNotValidException | GenericException | NotFoundException | AuthorizationDeniedException
+                | ValidationException | IOException | XmlException e) {
+                LOGGER.error("Error running premis skeleton on new representation: {}", e.getMessage());
+              }
             }
           }
         }
-      }
+      } else {
 
+        final Report reportItem = PluginHelper.initPluginReportItem(this, aip.getId(), AIP.class, AIPState.ACTIVE);
+        PluginHelper.updatePartialJobReport(this, model, reportItem, false, job);
+        reportState = PluginState.SKIPPED;
+        reportItem.setPluginState(PluginState.SKIPPED);
+        reportItem.setPluginDetails("This aip doens't have representations");
```

Comment:
spell check. Message sugestion: "Skipped because no representation was found for this AIP"